richardaspinall/poker-react-node
# feat: add player leave table backend logic

### Description

The backend logic for removing a player from a seat. I added very basic leave buttons in the frontend Seat file.

### How to test



1. Start up app
2. Click on a seat
3. Click on the "Leave Seat seat-number" link
4. Check console logs to confirm player has left seat

### Task

https://app.clickup.com/t/86cu6padm
CU-86cu6padm

### Checklist
